gm4_rope_ladders/data/gm4_rope_ladders/functions/init.mcfunction

```diff
@@ -0,0 +1,11 @@
+scoreboard objectives add gm4_rol_data dummy
+scoreboard objectives add gm4_rol_break_ladder minecraft.mined:minecraft.ladder
```

Comment:
Using `rol` (an abbreviation) isn't necessarily bad, but just to make sure you're aware of it:
In Minecraft 1.18 scoreboard objectives don't have an explicit length limit anymore, so feel free to use `gm4_rope_ladders_data`.